src/Explorer/Tabs/QueryTab/ResultsView.tsx

```diff
@@ -47,6 +48,52 @@ const ResultsTab: React.FC<ResultsViewProps> = ({ queryResults, isMongoDB, execu
     await executeQueryDocumentsPage(firstItemIndex + itemCount - 1);
   };
 
+  const ExportResults: React.FC = () => {
+    const [exportFormat, setExportFormat] = useState<"csv" | "json">("json");
+
+    const handleExport = (): void => {
+      if (exportFormat === "csv") {
+        const csvData = queryResults.documents.map((doc) => Object.values(doc).join(",")).join("\n");
+        const csvHeader = Object.keys(queryResults.documents[0]).join(",") + "\n";
```

Comment:
`,` separator doesn't work properly Excel and different locales (such as German) and even less with other apps, we should have a separator setting (see what we have done for VS Code: https://github.com/microsoft/vscode-cosmosdb/pull/2627) and use `;` as default (which is more compatible with other apps like LibreOffice and Apple Numbers) or add a separator header if we want to support Excel only (first line header line in file should be `sep=,` which is only Compatible with Excel).

Reply:
Just to confirm, are we sure we want to support Excel only? I'd rather use `;` as separator by default which works for Apple Numbers and LibreOffice on other platforms, maybe let's double check with the PM team?